packages/seacas/libraries/ioss/src/exodus/Ioex_BaseDatabaseIO.h

```diff
@@ -90,6 +90,94 @@ namespace Ioex {
     // database supports that type (e.g. return_value & Ioss::FACESET)
     IOSS_NODISCARD unsigned entity_field_support() const override;
 
+    IOSS_NODISCARD std::string get_internal_change_set_name() const override { return m_groupName; }
+
+    /** \brief Checks if a database type supports groups
+     *
+     *  \returns True if successful.
+     */
+    bool supports_group()
```

Comment:
Related to comment above -- is there a need to acquire a lock for this function

Reply:
Probably not but let me know what you'd like to do